circuits/circom/test/circuit.test.ts

```diff
@@ -0,0 +1,108 @@
+import { Point, sign } from '@noble/secp256k1'
+import { wasm as wasm_tester } from 'circom_tester'
+import { buildPoseidon } from 'circomlibjs'
+import { BigNumber, utils } from 'ethers'
+import { Scalar } from 'ffjavascript'
+
+import { join } from 'path'
+import {
+  bigintToArray,
+  bigintToUint8Array,
+  uint8ArrayToBigint,
+} from './helpers'
+
+// FIXME: unused?
+exports.p = Scalar.fromString(
+  '21888242871839275222246405745257275088548364400416034343698204186575808495617',
+)
+
+describe('Poseidon Merkle Tree', function () {
+  let poseidon
+  let F
+  let circuit
+
+  this.timeout(1000000)
+
+  before(async () => {
+    let p = join(__dirname, 'merkle_tree_test.circom')
```

Comment:
use `const` instead (`p` is not reassigned)
You'll see this warning by running
`pnpm run lint.check`
or `lerna run lint.check --scope=poseidon-ecdsa-membership` (in top level folder))